acai-ios/CollectionViewCells/CheckoutMenuItemCollectionViewCell.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  CheckoutMenuItemCollectionViewCell.swift
+//  acai-ios
+//
+//  Created by Artesia Ko on 3/12/19.
+//  Copyright © 2019 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class CheckoutMenuItemCollectionViewCell: UICollectionViewCell {
+    
+    var menuItemTitleLabel: UILabel!
+    var menuItemPriceLabel: UILabel!
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        
+        menuItemTitleLabel = UILabel()
+        menuItemTitleLabel.translatesAutoresizingMaskIntoConstraints = false
+        menuItemTitleLabel.text = ""
+        menuItemTitleLabel.font = Acai.avenirNextMedium
+        menuItemTitleLabel.textColor = .black
+        self.addSubview(menuItemTitleLabel)
```

Comment:
when adding subviews in cells, you should add it as a subview to the `contentView` instead of just `self`